packages/serial/src/serialapi/transport/SendDataBridgeMessages.test.ts

```diff
@@ -0,0 +1,206 @@
+import { TransmitStatus } from "@zwave-js/core";
+import { describe, expect, test } from "vitest";
+import { SendDataBridgeRequestTransmitReport } from "./SendDataBridgeMessages.js";
+
+describe("SendDataBridgeRequestTransmitReport", () => {
+	test("should not show TX report fields when transmitStatus is Fail", () => {
+		const rawMessage = new Uint8Array([
+			80,
+			TransmitStatus.Fail,
+			0,
+			0,
+			0,
+			0,
+			0,
+			0,
+			0,
+			0,
+			0,
+			0,
+			0,
+			0,
+			0,
+			0,
+			0,
+			0,
+			0,
+			0,
+			0,
+		]);
+		const report = SendDataBridgeRequestTransmitReport.from(
+			{ payload: rawMessage } as any,
+			{} as any,
+		);
```

Comment:
I feel like it would be easier to understand to use the `SendDataBridgeRequestTransmitReport` constructor, like so:
```ts
const report = new SendDataBridgeRequestTransmitReport({
  callbackId: ...,
  transmitStatus: TransmitStatus.Fail,
  // ... other fields
});
```

same for the other test cases.